### ISSUE TITLE
How to escape handlebars expressions?

### ISSUE THREAD

**Issue by adriano-di-giovanni** [Huachao/vscode-restclient#947]
I have strings that contain handlebars expressions, as follows

```http
POST /templates
Content-Type: application/json

{
  "text": "{{ping}}"
}
```

How do I escape `{{ping}}` to avoid restclient treating it as a variable?

Thanks

**Comment by Huachao** [comment]
@adriano-di-giovanni nice catch, it seems no work around for now, but if you don't define a variable named ping, the request body will still be as is.

**Comment by adriano-di-giovanni** [comment]
@Huachao, nope. The extension requires the variable to be defined. I escaped the handlebars extension as follows

```http
POST /templates
Content-Type: application/json

{
  "text": "\u007b\u007bping\u007d\u007d"
}
```

**Comment by Huachao** [comment]
@adriano-di-giovanni If not defined, it just shows an error and won't break the outgoing request

**Comment by vsnthdev** [comment]
I'm also having to send Handlebars strings in my payload and VSCode says there's an error. Thanks @adriano-di-giovanni for a workaround, I'll be also doing the same until a proper solution is implemented.
